Automattic/wp-calypso
# Update social-previews package to use icons as data URIs

## Proposed Changes

This PR updates the `social-previews` package to avoid relative import of `.png` icons and use the icons as data URIs instead.

This is to avoid bundling/import issues in the consumer code like in this case in Jetpack - https://github.com/Automattic/jetpack/actions/runs/4890874264/jobs/8730751438?pr=30352

## Testing Instructions

- Checkout this PR or use the live link below
- Goto `/marketing/traffic/:site` for a self hosted or a JN site.
- Scroll down to "Website Meta" section
- Click on "Show Previews"
- Confirm that the icons in the Facebook preview are same as before

## Pre-merge Checklist
